### ISSUE TITLE
componentWillReceiveProps don't receive updated values

### ISSUE THREAD

**Issue by OscarCanek** [radekmie/react-timepicker#5]
I wrapped the Timepicker in the following class:

```
import React, { Component } from 'react';
import Timepicker from 'react-timepicker';
import 'react-timepicker/timepicker.css';

class AwesomeTimePicker extends Component {
    constructor(props) {
        super(props);

        var time = this.props.defaultValue ? this.props.defaultValue.split(':') : [0, 0];

        this.state = {
            hours: parseInt(time[0]),
            minutes: parseInt(time[1])
        }
    }

    componentWillReceiveProps(nextProps) {
        var time = nextProps.defaultValue ? nextProps.defaultValue.split(':') : [0, 0];
        this.setState({
            hours: parseInt(time[0]),
            minutes: parseInt(time[1])
        });
    }

    onChangeTimePicker(hours, minutes) {
        if (hours && (hours === undefined || hours === null)) {
            hours = 0;
        }

        if (minutes && (minutes === undefined || minutes === null)) {
            hours = 0;
        }

        this.props.handleValue(this.props.name, hours + ":" + minutes);
    }

    render() {
        console.log(this.state.hours, this.state.minutes);
        const newProps = {
            hours: this.state.hours,
            minutes: this.state.minutes,
            onChange: this.onChangeTimePicker.bind(this),
            radius: 110
        }
        return (
            <Timepicker radius={110} hours={this.state.hours} minutes={this.state.minutes} onChange={this.onChangeTimePicker.bind(this)} />
        );
    }
}

export default AwesomeTimePicker;
```
But when the hours or minutes properties change the timepicker does not update the values.

I've been debuging and I found that the componentWillReceiveProps method don't receive the updated values when my render method call it.

**Comment by radekmie** [comment]
Hi, @OscarCanek. Yep, it's not handled. I'll fix it later today.

**Comment by radekmie** [comment]
Fixed in v1.3.1.
